roadmap/2022.md

```diff
@@ -0,0 +1,57 @@
+# Roadmap 2022
+
+This is the roadmap planned by the Integration team at MeiliSearch for the year 2022.
+
+Nothing is definitive: the roadmap can be adapted depending on the need of the users and the company.
+
+## 2022: the Main Lines
+
+#### Projects
+
+- Better know our users and our community to ease our decision making
+  - set the analytics on every integration
+  - extract information from then to prioritize our tasks
+- Keep working with the community, improve our way of working with the contributors by creating clearer issues, and by providing more and more good first issues.
+- Work with the devrel team to have help for the support for answering issues. The goal is to protect the developers of the integration team and make them focusing on the PR reviews and their coding tasks, rather than doing support.
```

Comment:
I'm just asking myself whether we could write @meilisearch/devrel-team on the markdown to refer to the devrel team. Will this change anything in the preview that Github does?

Reply:
It will only display `@meilisearch/devrel-team ` instead of `devrel team` :)
I can do it if you prefer

Reply:
Could be cool, WDYT?